rero_ils/modules/templates/jsonschemas/templates/template-v0.0.1.json

```diff
@@ -0,0 +1,136 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Template",
+  "description": "JSON schema for templates.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "template_type",
+    "name",
+    "creator",
+    "organisation",
+    "visibility"
```

Comment:
I think `data` field should be required, even if content is empty

Reply:
I will add this to the next PR